wordpress-mobile/WordPress-Android
# Feature/update refresh helper

### Fix
Update the `SwipeToRefreshHelper` class including:
- Deprecate old [`SwipeToRefreshHelper` constructor and initialization methods](https://github.com/wordpress-mobile/WordPress-Utils-Android/blob/develop/WordPressUtils/src/main/java/org/wordpress/android/util/helpers/SwipeToRefreshHelper.java#L21-L34) 
- Add new [`SwipeToRefreshHelper` constructor and initialization methods](https://github.com/wordpress-mobile/WordPress-Android/commit/e56f1e25a6dee2fc75a341057154780659352d6c).
- Add ability to use custom refresh colors like [SwipeRefreshLayout.setColorSchemeResources](https://developer.android.com/reference/android/support/v4/widget/SwipeRefreshLayout.html#setColorSchemeResources(int...)) method.
- Add `blue_wordpress` (`#0087be`) and `orange_jazzy` (`#f0821e`) colors to `SwipeToRefreshHelper` usages.

### Test
1. Go to any screen with swipe-to-refresh layout (e.g. ***Stats***, ***Reader***, ***Notifications***).
2. Swipe down to refresh content.
3. Notice refresh colors are `blue_wordpress` (`#0087be`) and `orange_jazzy` (`#f0821e`).
